scripts/static-vdis

```diff
@@ -259,8 +261,8 @@ def call_backend_detach(driver, config):
         raise Exception("SM_BACKEND_FAILURE(%d, %s, %s)" % xml)
     xml_rpc = xmlrpc.client.loads(xml[1])
     try:
-        res = xml_rpc[0][0]['params']
-    except:
+        res = xml_rpc[0][0]['params']  # type: ignore
```

Comment:
whyshould this type be ignored?

Reply:
`type: ignore` is not a problem here, as it is already inside a bare `except` (originally) so anything that can go wrong is already accounted for with a fallback to use `xml_rpc[0][0]`.

But in case you like it more, I propose the only other thing that is a `NOP` like that:
converting it to a cast to a dict. That also does nothing at runtime and helps typing more.

Instead of raising questions, I think this makes things more clear for the reader.

I also replaced the `IndexError` (that I added) with `except Exception` in order to be safe.

Further reasons why type checking is not needed here:

Because we can assume that the code currently works, and I this has the least impact here.

It is also at the end of the function, so nothing really depends on this type being correct.

I think so because at this and many other locations in this file, `xmlrpclib` has been replaced by `xmlrpc.client`.

As far as I understood, this Python3 conversion has been tested, and
as far as I understood, and is in production use in XS8 on the master branch currently.